scarb/build.rs

```diff
@@ -81,6 +81,34 @@ fn download_core(rev: &str) {
         }
     }
 
+    let ensure_err = "\nUnable to ensure Core library and Cairo Compiler versions match!\
```

Comment:
this line + usinc `panic!` directly is quite verbose. You can easily go with https://doc.rust-lang.org/stable/std/result/enum.Result.html#method.expect and say _Could not read this_, _Could not get field Y_ etc.